bindings.go

```diff
@@ -737,6 +738,49 @@ func NewQuery(pattern []byte, lang *Language) (*Query, error) {
 	}
 
 	q := &Query{c: c}
+
+	for i := uint32(0); i < q.PatternCount(); i++ {
+		steps := q.PredicatesForPattern(i)
+		if len(steps) == 0 {
+			continue
+		}
+
+		if steps[0].Type != QueryPredicateStepTypeString {
+			panic("preidcates must begin with a literal value")
```

Comment:
`preidcates` -> `predicate`

Reply:
let's avoid `panic` and return an error here instead

Reply:
see below